xdsl/dialects/riscv.py

```diff
@@ -47,7 +47,7 @@ def print_parameter(self, printer: Printer) -> None:
         pass
 
 
-class RdRsRsOp(IRDLOperation, ABC):
+class RdRsRsOperation(IRDLOperation, ABC):
```

Comment:
I'm thinking that we can stick to `Operation` suffix for abstract base classes, and `Op` for classes we expect to construct

Reply:
I would say that as long as we remain coherent, it's fine.
For instance, should we have `BinaryOperation` instead of `BinaryOp`?

Reply:
I would argue that it would be a good change to make, yeah